sw/device/silicon_creator/rom/BUILD

```diff
@@ -206,7 +205,6 @@ opentitan_binary(
     linker_script = ":linker_script",
     deps = [
         ":rom_without_keys",
-        "//sw/device/silicon_creator/rom/keys/real/rsa",
```

Comment:
should we probably remove this target entirely? since there will only be one ROM now that keys are in OTP?

Reply:
ok to do in a follow up if that was already your plan (if so pls disregard my comment :) )

Reply:
Will do it in a follow up PR :)